doc/mgmt/Management Framework.md

```diff
@@ -167,7 +175,7 @@ Management framework is a SONiC application which is responsible for providing v
     2. Custom YANG models ([SONiC YANG](https://github.com/Azure/SONiC/blob/master/doc/mgmt/SONiC_YANG_Model_Guidelines.md))
     3. Industry-standard CLI / Cisco like CLI
```

Comment:
Does management framework allows sonic cli & ISCLI intertwined? -config made through sonic cli shows up in ISCLI viceversa?

Reply:
Yes, the framework allows it; but it depends on the command implementation. You can mix the CLI channels as long as they operate on same set of tables. So, it is better to stick to one.